devel/icu/Portfile

```diff
@@ -25,17 +27,19 @@ long_description \
     support for supplementary Unicode characters (needed for GB 18030 repertoire support).
 
 homepage                    http://site.icu-project.org/
-master_sites    http://download.icu-project.org/files/${my_name}/${version}/
+master_sites    https://github.com/unicode-org/icu/archive/
+
+distname        release-[join [split ${version} .] -]
+
+checksums                   rmd160  2fbb53939b0945e908e9f85c25d47745171bcb35 \
+                            sha256  155a01d0e3d8443bb38c7d07fa8b7e3d6f3a5cfc5be810fe1632d76b4ca83f5a \
+                            size    44605077
 
-distname        ${my_name}-[join [split ${version} .] _]
-extract.suffix              .tgz
-distfiles                   ${distname}-src${extract.suffix}
-checksums                   rmd160  df06e7b18a87e383d3762564f2e9a59fd75865f9 \
-                            sha256  2b0a4410153a9b20de0e20c7d8b66049a72aef244b53683d0d7521371683da0c
+configure.python            /usr/bin/python
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    configure.pre_args      --prefix=${prefix}/libexec/libcxx-bootstrap
+    prefix      ${prefix}/libexec/libcxx-bootstrap
```

Comment:
Changing prefix will prevent the use of binaries.

Reply:
This should now be fixed.

Reply:
Thanks.